src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -11,7 +11,7 @@
   <PropertyGroup>
     <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' == ''">SR.PlatformNotSupported_NetHttp</GeneratePlatformNotSupportedAssemblyMessage>
-    <FeatureWasmThreads Condition="'$(TargetPlatformIdentifier)' == 'browser' and '$(MonoWasmBuildVariant)' == 'multithread'">true</FeatureWasmThreads>
+    <FeatureWasmThreads Condition="'$(TargetPlatformIdentifier)' == 'browser' and '$(WasmEnableThreads)' == 'true'">true</FeatureWasmThreads>
```

Comment:
Do we still need the `FeatureWasmThreads`? Could it be `WasmEnableThreads` instead? Are they semantically different?

Reply:
It is public, we have it in docs:
https://github.com/dotnet/runtime/blob/cd460dbda99c080b9f9ec1cbdcd233f25ba8c189/src/mono/wasm/threads.md?plain=1#L14

I don't think I can just remove it.

Reply:
That means it's public in the scope of the libraries, doesn't it?

Reply:
Okay, it could be changed but there's one advantage of keeping `FeatureWasmThreads`. `FeatureWasmThreads` is true only if both: `WasmEnableThreads` and `$(TargetPlatformIdentifier) == 'browser'` are true. If we drop `FeatureWasmThreads` then we need to change it into double condition.
The name is not intuitive, though. It should be `WasmEnableThreadsOnBrowser`.
@pavelsavara, what do you think?

Reply:
There is also `FeatureWasmPerfTracing` which triggers for the same condition now, but I think we could further differentiate the two.

So I would keep `FeatureWasmThreads` separate for the same reason. The actual meaning is `FeatureWasmManagedThreads` is you want to rename it.

Reply:
Thanks. The best solution then: keep the property and rename to `FeatureWasmManagedThreads`

Reply:
Please give me some time before you resolve this with best solution. I'm trying (offline) to understand why we need these distinct properties and why are they defined in every library

Reply:
I'm good, thanks!